.github/workflows/push-foundation-repo.yml

```diff
@@ -20,12 +23,15 @@ jobs:
           token: ${{ secrets.SYNCAZF }}
 
       - name: Push to Aleph-Zero-Foundation
+        # yamllint disable-line rule:line-length
         run: git push https://x-access-token:${{ secrets.SYNCAZF }}@github.com/aleph-zero-foundation/aleph-node.git
 
 
   sync-release-branch:
     runs-on: ubuntu-20.04
-    if: github.repository == 'Cardinal-Cryptography/aleph-node' && startsWith(github.ref, 'refs/heads/') && startsWith(github.ref_name, 'release-')
+    if: >
```

Comment:
Let's make this `if` in a format of:
````
if: >
  something1(a, b, c)
  && something2(d, e, f)
  && something3(g, h, i)
````
where the conditions are not cut in half or something.  Whether `&&` is in the beginning or at the end - it doesn't matter I guess, pick one.